source/DSCResources/Resources/windows.ProcessMitigation.ps1

```diff
@@ -25,7 +25,15 @@ foreach ($target in $mitigationTargets)
         $idValue += $rule.id
     }
 
-    $enableValue = $enableValue.split(',')
+    if($enableValue.count -gt 1)
```

Comment:
style guidelines wants a space after the keyword "if" and before the conditional.